src/triggers.c

```diff
@@ -938,6 +938,17 @@ void hurt_touch()
 {
 	if (!other->s.v.takedamage)
 	{
+		if (cvar("k_ctf_hurt_items"))
+		{
+			if (streq(other->classname, "item_flag_team1") || streq(other->classname, "item_flag_team2"))
```

Comment:
For some reason this does not work when a flag carrier falls into the trigger_hurt and is gibbed. The rune is correctly destroyed, but the flag doesn't trigger hurt_touch. If I comment out SOLID_NOT further down this function (which would break periodic damage), the hurt_touch function is correctly called for `item_flag_team(1|2)` ... any idea on what a proper solution might be? An alternative would be to have two triggers on the map. One that kills the player, and one further down that kills the flag. This also fixes the problem, but would require ent file or map modifications.

Throwing the flag down into the trigger works correctly however.